app/assets/stylesheets/application.css.scss

```diff
@@ -13,3 +13,14 @@
  *= require_tree .
  *= require_self
  */
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+
+#main-container {
+  margin-top: 100px;
```

Comment:
Properties should be ordered margin-left, margin-right, margin-top, padding-left, padding-right